app/jobs/set_linkset_home_job.rb

```diff
@@ -0,0 +1,14 @@
+class SetLinksetHomeJob < ApplicationJob
+  queue_as :default
+
+  def perform(version:)
+    return unless version.latest? && version.indexed?
+
+    gem = RubygemFs.instance.get("gems/#{version.gem_file_name}")
```

Comment:
This is the only job that redownloads the gem besides the content storage job. Could it just grab this info on push?

Reply:
right now we don't pass anything besides the version itself to the "after write" set of jobs, so I kept in keeping with that